app/controllers/brexit_checker_controller.rb

```diff
@@ -9,16 +9,16 @@ class BrexitCheckerController < ApplicationController
   end
 
   def show
-    @questions = BrexitChecker::Question.load_all
-    @page_service = BrexitChecker::PageService.new(questions: @questions,
-                                                criteria_keys: criteria_keys,
-                                                current_page_from_params: page)
-
-    if @page_service.redirect_to_results?
-      redirect_to brexit_checker_results_path(c: criteria_keys)
-    else
-      @current_question = @questions[@page_service.current_page]
-    end
+    all_questions = BrexitChecker::Question.load_all
```

Comment:
Minor thing: I was slightly surprising to see this here, mostly because we don't inject it in [other places](https://github.com/alphagov/finder-frontend/blob/master/spec/lib/brexit_checker/criteria/filter_spec.rb#L34). I'm wondering if there's a particular reason for injecting it here?

Reply:
Not sure what you mean by "it" here? We need to load all the questions to pass them to the `next_question_index` method?